packages/crew_integration.rb

```diff
@@ -0,0 +1,78 @@
+require 'package'
+
+class Crew_integration < Package
+  description 'Improves integration with Chrome OS shell and Chromebrew GUI applications.'
+  homepage 'https://github.com/skycocker/chromebrew/crew_integration'
+  version '2.0'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  depends_on "graphicsmagick"
+    
+  case ARGV[0]
+  when 'reinstall', 'install'
+     FileUtils.rm_rf "#{CREW_PREFIX}/lib/pwa/"
+     FileUtils.rm_rf "#{HOME}/MyFiles/.extension"
```

Comment:
Should these lines be in the `def self.remove` section?

Reply:
Moved to `remove` section